src/model.py

```diff
@@ -208,10 +189,7 @@ def forward(self, x):
         # shape(x) = N, L, C
         # conv1d needs N,C,L
         x = x.transpose(1,2)
-        x = self.conv1(x)
-        x = self.conv2(x)
-        x = self.conv3(x)
-        #x = self.conv4(x)
+        x = self.conv_blocks(x)
```

Comment:
I miss this fix in the last request. Now is consistent with whats is defined above.